pkg/onboarding/presentation/rest/handlers.go

```diff
@@ -1214,3 +1217,22 @@ func (h *HandlersInterfacesImpl) CheckHasPermission() http.HandlerFunc {
 		serverutils.WriteJSONResponse(rw, nil, http.StatusOK)
 	}
 }
+
+// GetBearerTokenAuthorizationHeader get the authorization header
+func (h *HandlersInterfacesImpl) GetBearerTokenAuthorizationHeader() http.HandlerFunc {
```

Comment:
Are we getting the `authorizationHeader` or just the `JWT` or they are synonymous?

Reply:
synonymous